aws/protocol/query/encoder.go

```diff
@@ -0,0 +1,27 @@
+package query
+
+import "net/url"
+
+// Encoder is a Query encoder that supports construction of Query body
+// values using methods.
+type Encoder struct {
```

Comment:
Should the query encoder be a part of smithy-go?

Reply:
No reason for it to - it's an AWS specific protocol.